action.yaml

```diff
@@ -0,0 +1,16 @@
+name: 'Run pytest'
```

Comment:
So, is this file a result of the failed experiment of making `pytest-pyodide` a custom action?

Reply:
oops, yes. Deleted.